src/Packing.cpp

```diff
@@ -0,0 +1,67 @@
+//
+// Created by rolf on 03-03-21.
+//
+
+#include "Packing.h"
+#include "utilities.h"
+#include <cmath>
+
+RobotCommandPayload createLowLevelCommand(const proto::RobotCommand& proto,  const proto::World& world, bool isYellow){
+  float rho = sqrtf(proto.vel().x() * proto.vel().x() + proto.vel().y() * proto.vel().y());
+  float theta = atan2f(proto.vel().y(), proto.vel().x());
+  auto bot = rtt::robothub::utils::getWorldBot(proto.id(),isYellow,world);
+
+  RobotCommand command;
+  command.header = PACKET_TYPE_ROBOT_COMMAND;
+  command.id = proto.id();
+
+  command.doKick = proto.kicker();
+  command.doChip = proto.chipper();
+  command.doForce = proto.chip_kick_forced();
+
+  command.rho = rho;
+  command.theta = theta;
+  command.angle = proto.w();
+
+  if(bot){
+    command.useCameraAngle = true;
+    command.cameraAngle = bot->w();
+  }else{
+    command.useCameraAngle = false;
+    command.cameraAngle = 0.0;
+  }
+  command.kickChipPower = proto.chip_kick_vel();
+  command.dribbler = proto.dribbler();
+  command.angularControl = proto.use_angle();
```

Comment:
If this is true, should we do something with `omega`? I recall multiplying by 8

Reply:
Yes, but I probably wouldn't bother with this for now, since it's unused.